langchain4j-mongodb-atlas/src/main/java/dev/langchain4j/store/embedding/mongodb/MongoDbMatchedDocument.java

```diff
@@ -9,7 +9,7 @@ public class MongoDbMatchedDocument {
     private String id;
     private List<Float> embedding;
     private String text;
-    private Map<String, String> metadata;
+    private Map<String, Object> metadata;
```

Comment:
We wrote a utility to convert ObjectIds to strings within metadata, as there was some requirement that they wished metadata to be serialized to json. Just something to keep in mind for future.

Reply:
Could you link, just for reference?